demos/notebooks/demo_pipeline.ipynb

```diff
@@ -368,17 +555,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# cnm1 = cnmf.CNMF(n_processes, params=opts, dview=dview)\n",
-    "# cnm1.fit_file(motion_correct=True)"
+    "f, ((ax_max_orig, ax_max), (ax_corr_orig, ax_corr)) = plt.subplots(2,2,figsize=(6,6), sharex=True, sharey=True)\n",
+    "\n",
+    "ax_max_orig.imshow(max_projection_orig, \n",
+    "                   cmap='viridis', \n",
+    "                   vmin=np.percentile(np.ravel(max_projection_orig),50), \n",
+    "                   vmax=np.percentile(np.ravel(max_projection_orig),99.5));\n",
+    "ax_max_orig.set_title(\"Max Projection Orig\", fontsize=12);\n",
+    "ax_max.imshow(max_projection, \n",
+    "              cmap='viridis', \n",
+    "              vmin=np.percentile(np.ravel(max_projection),50), \n",
+    "              vmax=np.percentile(np.ravel(max_projection),99.5));\n",
+    "ax_max.set_title(\"Max Motion Corrected\", fontsize=12);\n",
+    "\n",
+    "ax_corr_orig.imshow(correlation_image_orig, \n",
+    "                    cmap='viridis', \n",
+    "                   vmin=np.percentile(np.ravel(correlation_image_orig),50), \n",
+    "                   vmax=np.percentile(np.ravel(correlation_image_orig),99.5));\n",
+    "ax_corr_orig.set_title('Correlation Image Orig', fontsize=12);\n",
+    "\n",
+    "\n",
+    "ax_corr.imshow(correlation_image, \n",
+    "               cmap='viridis', \n",
+    "               vmin=np.percentile(np.ravel(correlation_image),50), \n",
+    "               vmax=np.percentile(np.ravel(correlation_image),99.5));\n",
+    "ax_corr.set_title('Correlation Motion Corrected', fontsize=12);\n",
+    "\n",
+    "plt.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\" markdown=\"1\">\n",
+    "    <h2 style=\"margin-top: 0;\">If you don't need to run motion correction</h2>  \n",
+    "If you don't need to run motion correction  (for instance if your movie has no movement -- which is often the case in slice praparations), you can directly memory map your file to prepare for subsequent  processing:\n",
```

Comment:
do we want to suggest this? I wonder if people eyeballing it and deciding "no movement" might actually get it wrong and omit a step that may benefit them

Reply:
Also, there's an extra space between subsequent and processing

Reply:
This finishes out issue #1083 . We shouldn't enforce costly motion correction on people. Some people also use other tools for motion correction and just want caiman for cnmf. Others work in slice. Many use cases.

Space removed.

Reply:
yes I agree with Eric, each algo of caiman should be self-contained :smile: